pkg/syntheticmonitoring/synthetic-monitoring-handler.go

```diff
@@ -26,6 +26,7 @@ import (
  */
 
 const smBaseURL = "https://synthetic-monitoring-api.grafana.net"
+const smRegionURL = "https://synthetic-monitoring-api-%s.grafana.net"
```

Comment:
I would add the same exceptions as here: https://github.com/grafana/terraform-provider-grafana/blob/main/internal/resources/cloud/resource_synthetic_monitoring_installation.go#L16-L22

Otherwise, if someone creates a stack in `eu`, and tries to use Grizzly, it will fail. All new APIs are being provisioned with the API URL you have here

Reply:
I'm seeing that the list [here](https://grafana.com/docs/grafana-cloud/monitor-public-endpoints/set-up/set-up-private-probes/) is larger than the one set in TF provider.

Reply:
That's probes, not API endpoints, right? A probe is the remote node that pings your host. They connect to an API for configuration. There will be far fewer API endpoints than probes.